### ISSUE TITLE
Questions ,thx

### ISSUE THREAD

**Issue by goodsm0** [ForestsKing/GLAFF#7]
Hi,
I'm not quite clear about the following code. Why is it necessary to perform concatenation? Thank you!!
if self.args.flag == 'Plugin':
            loss = self.criterion(torch.concat([pred, reco, map1, map2], dim=1),
                                  torch.concat([y_data, x_data, y_data, y_data], dim=1))

**Comment by ForestsKing** [comment]
Thank you for your attention to our work. The concatenation was originally intended to explicitly constrain the various modules of GLAFF, ensuring they function as designed. However, in recent experiments, we found that explicit constraints are not necessary, so we have removed this part of the code.
